.hemtt/project.toml

```diff
@@ -0,0 +1,31 @@
+name = "Advanced Equipment"
+prefix = "ae3"
+mainprefix = "z"
```

Comment:
I can't find `mainprefix` in the HEMTT documentation. If I understand the documentation correctly `prefix` could be used as a variable in the `$PBOPREFIX$` files. Not sure.

Reply:
ACE3 does this the same way.